cmake/cpackEngineRPM.cmake

```diff
@@ -81,9 +81,9 @@ if (${REDHAT_VERSION_NUMBER} EQUAL 6)
     # Disable auto require as this will also try to pull Boost via RPM
     SET(CPACK_RPM_PACKAGE_AUTOREQPROV " no")
 elseif (${SUSE_VERSION_NUMBER} EQUAL 12)
-    SETA(CPACK_RPM_columnstore-engine_PACKAGE_REQUIRES "boost-devel >= 1.54.0" "libsnappy1" "jemalloc" "net-tools" "MariaDB-server" "python3")
+    SETA(CPACK_RPM_columnstore-engine_PACKAGE_REQUIRES "boost-devel >= 1.54.0" "libsnappy1" "net-tools" "MariaDB-server" "python3")
```

Comment:
Cpack doesn't use these files anymore. Actuall requirerenments for rpm are in the server's CMakeLists.txt

Reply:
I am aware of this but some of us still build separate and local. This was to speed up my own testing.